packages/components/pie-textarea/test/component/pie-textarea.spec.ts

```diff
@@ -183,97 +195,108 @@ test.describe('PieTextarea - Component tests', () => {
 
                 // Assert
-                expect((await component.locator('textarea').inputValue())).toBe('testValue');
+                expect((await textarea.inputValue())).toBe('testValue');
             });
         });
 
         test.describe('required', () => {
-            test('should not render the required property on the textarea element if no required property is provided', async ({ mount }) => {
+            test('should not render the required property on the textarea element if no required property is provided', async ({ page }) => {
                 // Arrange
-                const component = await mount(PieTextarea, {});
+                const textAreaPage = new BasePage(page, 'textarea');
+                await textAreaPage.load();
 
                 // Act
-                const textarea = component.locator('textarea');
+                const textarea = page.getByTestId(textArea.selectors.container.dataTestId);
 
                 // Assert
                 expect((await textarea.getAttribute('required'))).toBe(null);
             });
 
-            test('should apply the property property to the HTML textarea rendered', async ({ mount }) => {
+            test('should apply the property property to the HTML textarea rendered', async ({ page }) => {
                 // Arrange
-                const component = await mount(PieTextarea, {
-                    props: {
-                        required: true,
-                    } as TextareaProps,
-                });
+                const textAreaPage = new BasePage(page, 'textarea');
+                const props: Partial<TextareaProps> = {
+                    required: true,
+                };
+
+                await textAreaPage.load({ ...props });
 
                 // Act
-                const textarea = component.locator('textarea');
+                const textarea = page.getByTestId(textArea.selectors.container.dataTestId);
 
                 // Assert
                 expect((await textarea.getAttribute('required'))).toBe('');
             });
 
-            test('should be in an invalid `valueMissing` state if the textarea is empty and required property is set to `true`', async ({ mount, page }) => {
+            test('should be in an invalid `valueMissing` state if the textarea is empty and required property is set to `true`', async ({ page }) => {
                 // Arrange
-                await mount(PieTextarea, {
-                    props: {
-                        required: true,
-                    } as TextareaProps,
-                });
+                const textAreaPage = new BasePage(page, 'textarea');
+                const props: Partial<TextareaProps> = {
+                    required: true,
+                };
+
+                await textAreaPage.load({ ...props });
 
                 // Act
                 const isInvalid = await page.evaluate(() => document.querySelector('pie-textarea')?.validity.valueMissing);
```

Comment:
Leverage the approach mentioned above if possible. There's example in `pie-radio` i believe if you need a reference!